Library/Homebrew/formula_cellar_checks.rb

```diff
@@ -442,8 +444,9 @@ def relative_glob(dir, pattern)
     File.directory?(dir) ? Dir.chdir(dir) { Dir[pattern] } : []
   end
 
+  sig { params(file: T.any(Pathname, String), objdump: String).returns(T::Boolean) }
   def cpuid_instruction?(file, objdump = "objdump")
```

Comment:
```suggestion
  sig { params(file: T.any(Pathname, String), objdump: Pathname).returns(T::Boolean) }
  def cpuid_instruction?(file, objdump)
```

Reply:
The `Pathname` part is actually necessary since callers above pass `objdump` as a `Pathname`.